webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig

```diff
@@ -1,19 +1,38 @@
-<div class="row">
-    <div class="col-lg-4">
-        {{ form_start(form) }}
-        {% if form.offsetExists('externalid') %}
-            {{ form_row(form.externalid) }}
-        {% endif %}
-        {{ form_row(form.shortname) }}
-        {{ form_row(form.name) }}
-        <div class="form-group">
-            {{ form_label(form.country) }}
-            {{ form_widget(form.country) }}
-            <small class="form-text text-muted">
-                <a target="_blank" href="http://en.wikipedia.org/wiki/ISO_3166-1_alpha-3#Current_codes"><i class="fas fa-question-circle"></i></a>
-            </small>
-        </div>
-        {{ form_row(form.comments) }}
-        {{ form_end(form) }}
-    </div>
+<div class="row col-lg-4">
+    {{ form(form) }}
 </div>
+
+<style>
+    img.countryflag {
+        margin-top: -3px;
+    }
+</style>
+
+<script>
+    var template = function(element) {
+        var imgBase = '';
+        {% set countryFlag = 'images/countries/NLD.png' %}
+        {% if countryFlag | assetExists %}
+            // Use a country flag we know exists to automatically determine base path and asset version
+            imgBase = "{{ asset(countryFlag) }}";
+        {% endif %}
+
+        var img = element.id == undefined || element.id == "" || imgBase == ""
+            ? ''
+            : '<img src="' + imgBase.replace("NLD.png", element.id + ".png", 1) + '" class="countryflag">&nbsp;';
+
+        return img + element.text;
+    };
+
+    $('#team_affiliation_country').select2({
+        templateResult: template,
+        templateSelection: template,
+
+        // Prevent escaping
```

Comment:
why is this necessary?

Reply:
Otherwise you see `<img...` instead of an image